impeller/entity/entity_pass.cc

```diff
@@ -256,13 +256,15 @@ static EntityPassTarget CreateRenderTarget(ContentContext& renderer,
   /// What's important is the `StorageMode` of the textures, which cannot be
   /// changed for the lifetime of the textures.
 
-  if (context->GetBackendType() != Context::BackendType::kMetal) {
-    // TODO(https://github.com/flutter/flutter/issues/141495): Implement mip map
-    // generation on vulkan.
-    // TODO(https://github.com/flutter/flutter/issues/141732): Implement mip map
-    // generation on opengles.
-    mip_count = 1;
-  }
+  // if (context->GetBackendType() != Context::BackendType::kMetal) {
```

Comment:
friendly note, switch this to mip_count 1 for opengles until we can verify that is working.